apps/daffio/src/app/core/sidebar/containers/sidebar-viewport/sidebar-viewport.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import { Observable } from 'rxjs';
+
+import { ToggleSidebar, CloseSidebar, OpenSidebar, SetSidebarState } from '../../actions/sidebar.actions';
+import * as fromDaffioSidebar from '../../reducers/index';
+
+@Component({
+  selector: 'daff-sidebar-viewport-container',
+  templateUrl: './sidebar-viewport.component.html'
+})
+export class DaffSidebarViewportContainer {
```

Comment:
this should be named `SidebarViewportContainer`

Reply:
resolved in 995ad4c